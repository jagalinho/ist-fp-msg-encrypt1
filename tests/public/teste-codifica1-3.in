chave = (('a', 'b', 'c', 'd', 'e'), ('f', 'g', 'h', 'i', 'j'), ('k', 'l', 'm', 'n', 'o'), ('p', 'q', 'r', 's', 't'), ('u', 'w', 'v', 'x', 'z'))
cad = 'whatisthecodedphrase'
cad_encriptada = codifica1(cad, chave)
cad_encriptada
