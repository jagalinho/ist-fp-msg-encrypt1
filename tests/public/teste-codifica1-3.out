'4112003413333412040224030403301232003304'
